client/script.js

```diff
@@ -0,0 +1,532 @@
+const root = document.getElementById('root');
+
+
+// CREATING NAV BAR
+const nav = ['Travel updates', 'Reviews', 'About', 'Contact'];
+
+function createNav(nav) {
+    const navBar = document.createElement('nav');
+    navBar.setAttribute('class', 'nav');
+
+    const ul = document.createElement('ul');
+    ul.setAttribute('class', 'nav__container');
+    navBar.appendChild(ul);
+
+    nav.forEach(element => {
+        const li = document.createElement('li');
+        li.setAttribute('class', 'nav__item');
+        const anchor = document.createElement('a');
+        anchor.setAttribute('href', '#/');
+        anchor.setAttribute('class', 'nav__link');
+        anchor.textContent = element;
+
+        ul.appendChild(li);
+        li.appendChild(anchor);
+
+    })
+
+    return navBar;
+}
+

```

Comment:
I would have render the navbar in the previous function to make the code easier to read